.env.j2

```diff
@@ -0,0 +1,8 @@
+FLASK_APP=todo_app/app
+FLASK_DEBUG=true
+SECRET_KEY=secret-key
+TRELLO_API_TOKEN={{ trello_api_token }}
+TRELLO_API_KEY={{ trello_api_key }}
+BOARD_ID=65cb8fae819b942721d5d651
+NOT_STARTED_LIST_ID=65cca50aa8293b273d55645f
+DONE_LIST_ID=65ccb02ee5309ff989057b63
```

Comment:
Very minor - I'd do the same for this exercise but if this were a real project I'd recommend pulling these out into variables in your playbook, even if we keep the values saved inside that as a default. They're not sensitive so we don't need to hide them, but we would likely want to eventually swap them out for a new environment
(Wouldn't bother changing here, but worth noting)